modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -52,6 +52,10 @@ func NewKeeper(
 		legacySubspace = legacySubspace.WithKeyTable(types.ParamKeyTable())
 	}
 
+	if strings.TrimSpace(authority) == "" {
+		panic(fmt.Errorf("authority must be non-empty"))
```

Comment:
Was it really needed to use `Errorf`, since the string does not require formatting?